### PR TITLE
Added MongoDB regex QC to check if validator statement is part of db.…

### PR DESCRIPTION
**Pull request by lbjenn**
…createCollection call in mongosh includes a validator or an error is raised

### CODE REVIEW

File: MongoDB/ mongoCrCollectionValidatorChk.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+            "changeSet": {
+                "id": "create_students-collection",
+                "author": "jennl",
+                "labels": "[jira-101],'version 1.0'",
+                "runWith": "mongosh",
+                "changes": [
+                  {
+                    "mongoFile": {
+                      "dbms": "mongodb",
+                      "path": "create-collection-students.js",
```
Comment (molivasdat) [inline MongoDB/ mongoCrCollectionValidatorChk.md:75]: create-collection-students-fail.js to make it more obvious

File: MongoDB/ mongoCrCollectionValidatorChk.md [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+                "rollback": [
+                    {
+                      "mongo": {
+                        "mongo": "db.students.drop()"
+                      }
+                    }
+                ]
+            }    
+        }
+--create-collection-students.js contents
```
Comment (molivasdat) [inline MongoDB/ mongoCrCollectionValidatorChk.md:89]: Change to create-collection-students-fail.js to make it more obvious

**Review by molivasdat — CHANGES_REQUESTED** [review]
No need for the .DS_Store file, please remove it from the PR.

File: MongoDB/ mongoCrCollectionValidatorChk.md (hunk 1 above)
Comment (lbjenn) [reply MongoDB/ mongoCrCollectionValidatorChk.md]: I have completed this and pushed it to repo

**Review by molivasdat — APPROVED** [review]
New changes look good.

**Pull request merged**